### PR TITLE
dist/tools/ethos: deduplicate setup_network.sh, start_network.sh

### PR DESCRIPTION
**Pull request by kaspar030**
### Contribution description

#11816 introduced setup_network.sh, introducing some duplication with start_network.sh.
This PR aims to remove the duplications.

### Testing procedure

- ensure examples/suit_update (#11818) still works (using setup_network.sh)
- ensure users of start_network.sh still work

### CODE REVIEW

**Comment by fjmolinas** [comment]
ping @kaspar030

**Comment by fjmolinas** [comment]
ping @kaspar030

File: dist/tools/ethos/start_network.sh [hunk 1]
```diff
@@ -2,31 +2,14 @@
...
-    kill ${UHCPD_PID}
+    [ -n "${SETUP_NETWORK_PID}" ] && kill -TERM ${SETUP_NETWORK_PID}
     trap "" INT QUIT TERM EXIT
 }
 
-start_uhcpd() {
-    ${UHCPD} ${TAP} ${PREFIX} > /dev/null &
-    UHCPD_PID=$!
+setup_network() {
+    ${ETHOS_DIR}/setup_network.sh $1 $2 &
```
Comment (miri64) [inline dist/tools/ethos/start_network.sh]: In the spirit of future proofness:
```suggestion
    ${ETHOS_DIR}/setup_network.sh $* &
```

So later extensions to the script do not need this function to be touched.

**Comment by fjmolinas** [comment]
ping @kaspar030!

**Comment by fjmolinas** [comment]
this should be a low hanging fruit..

**Comment by miri64** [comment]
Ping @kaspar030

**Comment by kaspar030** [comment]
> this should be a low hanging fruit..

honestly I'm lost on this one, rebasing is messy due to the DHCPv6 additions. Does one of you want to take over (or re-do it)?

**Comment by miri64** [comment]
> > this should be a low hanging fruit..
> 
> honestly I'm lost on this one, rebasing is messy due to the DHCPv6 additions. Does one of you want to take over (or re-do it)?

Was planning on that anyway (also to integrate with `tapsetup`) some time. Let's close this as memo for now then?

**Pull request closed without merging**